ast.go

```diff
@@ -91,6 +113,10 @@ type WriteStatement interface {
 	iWriteStatement()
 	iStatement()
 	GetTable() *Table
+
+	// Resolve returns a string representation with custom function nodes resolved to the values
+	// passed by resolver.
+	Resolve(WriteStatementResolver) (string, error)
```

Comment:
All `WriteStatement`s must implement this method. Those statements are `Insert`, `Update`, and `Delete`